mycli/main.py

```diff
@@ -91,6 +91,8 @@ class MyCli(object):
 
     system_config_files = [
         '/etc/myclirc',
+        os.path.join(
+            os.environ.get("XDG_CONFIG_HOME", "~/.config"), "mycli", "myclirc")
     ]
```

Comment:
Is it necessary to check if the `XDG_CONFIG_HOME` is valide?

For example, what if the env exist but is empty string?

Reply:
Yes, it has to be checked. Thanks for your comment.
I'll push a new commit.

Reply:
And, I don't think we don't need to check if the `XDG_CONFIG_HOME` is REALLY valid or not.
In the latest commit 8aa2466 , I added a code to check just it is empty or not.
However, when it exists and not empty, even if the referred path is NOT valid (i.e., path not exists), I think we should respect they set the very path for the envvar.